components/server/src/session-handler.ts

```diff
@@ -124,6 +124,23 @@ export class SessionHandlerProvider {
         }
     }
 
+    public async createJWTSessionCookie(
+        userID: string,
+    ): Promise<{ name: string; value: string; opts: express.CookieOptions }> {
+        const token = await this.authJWT.sign(userID, {});
+
+        return {
+            name: SessionHandler.getJWTCookieName(this.config),
+            value: token,
+            opts: {
+                maxAge: this.config.auth.session.cookie.maxAge * 1000, // express does not match the HTTP spec and uses milliseconds
+                httpOnly: this.config.auth.session.cookie.httpOnly,
+                sameSite: this.config.auth.session.cookie.sameSite,
```

Comment:
Somehow seeing this being configurable makes me uneasy... :worried: But I recall we discussed before, and I assume we had good reasons. :ok_hand:

Reply:
The cookie config has to be consistent between public-api and server. That means the only way to ensure that is that we share the same config. It's not configurable from the `installer` side, but both public-api and server components use the same underlying config